apollo-router/src/plugins/authentication/mod.rs

```diff
@@ -128,6 +128,9 @@ struct JWTConf {
     /// Header value prefix
     #[serde(default = "default_header_value_prefix")]
     header_value_prefix: String,
+    /// Whether to ignore any mismatched prefixes
+    #[serde(default)]
+    ignore_other_prefixes: bool,
```

Comment:
I'm not sure we actually need an option. The router looks for the value with the prefix it needs, it ignores the rest, we still have the same behavior as before: if it does not find the value it wants, then the request will not be authenticated

Reply:
the important part is whether `require_authentication` is enabled